src/Service/PetTypeService.php

```diff
@@ -46,9 +47,10 @@ public function findAll(): iterable
         ]);
     }
 
-    public function update(string $id, PetType $petType): void
+    public function update(PetType $petType, stdClass $request): void
```

Comment:
use `Request $request` here, because $request is a `Symfony\Component\HttpFoundation\Request` instance